pkg/db/relying_parties.go

```diff
@@ -9,17 +9,20 @@ package db
 import (
 	"database/sql"
 	"fmt"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
 )
 
 const (
-	sqlInsertRelyingParty         = `insert into relying_party (client_id) values (?)`
+	sqlInsertRelyingParty         = `insert into relying_party (client_id, did) values (?, ?)`
 	sqlRelyingPartyFindByClientID = `select * from relying_party where client_id = ?`
 )
 
 // RelyingParty represents the relying party.
 type RelyingParty struct {
 	ID       int64
 	ClientID string
+	DID      *did.DID
```

Comment:
For RP, it should normally a OIDC flow. RP shouldn't be aware of DID/DIDComm, as per my understanding. Are we changing that assumption now ?

Reply:
This attribute is simply the RP's public DID. This PR assumes the RP's public DID is registered in the adapter.

Reply:
This PR then delivers the RP's public DID to the wallet, from which did-exchange may commence.